### PR TITLE
Create tgruenewald-boulder.md

### PR DESCRIPTION
**Pull request by tgruenewald**
i added some restaurants and things to do.

:tennis:

### CODE REVIEW

**Comment by MichaelRye** [comment]
Insert comment text here. :)

File: tgruenewald-boulder.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
+### Boulder, CO
+#### Food
+Pizzaria Locale,
```
Comment (carolynshin) [inline tgruenewald-boulder.md:4]: @tgruenewald, I :heart: Pizzaria Locale!!!

**Comment by carolynshin** [comment]
@tgruenewald, great recommendations!

**Comment by tgruenewald** [comment]
Thanks @carolynshin !!

**Pull request merged**